src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -466,10 +466,17 @@ const OfflineTokenRow = ({ offlineToken, onValueChanged, validationFailed }) =>
     const validationStateToken = validationFailed.offlineToken ? 'error' : 'default';
 
     return (
-        <FormGroup label={_("Offline Token")} fieldId="offline-token"
+        <FormGroup label={_("Offline token")} fieldId="offline-token"
                    id="offline-token-group"
                    helperTextInvalid={validationFailed.offlineToken}
-                   validated={validationStateToken}>
+                   validated={validationStateToken}
+                   helperText={<span>
+                       <a href="https://access.redhat.com/management/api" target="_blank" rel="noopener noreferrer">
+                           <ExternalLinkAltIcon className="pf-u-mr-xs" />
+                           {_("Get a new RHSM token")}
+                       </a>
+                       {_(", then copy the token and paste it above.")}
```

Comment:
This is another case of stringing together two half-sentences and making assumptions about English. For simplicity, let's just make this two separate sentences.